skimage/novice/__init__.py

```diff
@@ -29,7 +29,7 @@
 
 We can display pictures (after running this command, close the window to access the prompt again):
 
->>> picture.show()
+>>> picture.show()  # doctest: +SKIP
```

Comment:
These were needed to prevent the build from hanging, since we can no longer install `python-tk`.